web/src/components/roadmap/SidebarCardsList.tsx

```diff
@@ -0,0 +1,154 @@
+            <Accordion
+              elevation={0}
+              expanded={hiddenAccordionIsOpen}
+              onChange={() => setHiddenAccordionIsOpen(!hiddenAccordionIsOpen)}
+              sx={{
+                backgroundColor: `#F9FBFB`,
+              }}
+            >
+              <AccordionSummary
+                expandIcon={<Icon className="usa-icon--size-5 margin-x-1 text-normal">expand_more</Icon>}
+                aria-controls="hidden-opportunity-content"
+                id="hidden-opportunity-header"
+                data-testid="hidden-opportunity-header"
+              >
+                <div className="margin-y-2">
+                  <div className="flex flex-align-center margin-0-override text-normal">
+                    <div className="inline">
+                      {templateEval(Config.dashboardDefaults.hiddenOpportunitiesHeader, {
+                        count: String(hiddenOpportunitiesCount()),
+                      })}
+                    </div>
+                  </div>
+                </div>
+              </AccordionSummary>
+              <AccordionDetails data-testid="hidden-opportunities">
+                {hiddenSortedCertifications.map((cert) => (
+                  <OpportunityCard key={cert.id} opportunity={cert} urlPath="certification" />
+                ))}
+                {/* {hiddenSortedFundings.map((funding) => (
+            <OpportunityCard key={funding.id} opportunity={funding} urlPath="funding" />
+          ))} */}
+              </AccordionDetails>
+            </Accordion>
+          </div>
+          <hr className="desktop:margin-right-1 margin-bottom-3 bg-roadmap-blue-light" aria-hidden={true} />
+        </>
+      );
+    } else {
+      return <></>;
+    }
+  };
+
+  return (
+    <>
+      <hr
+        className="desktop:margin-right-1 margin-top-2 margin-bottom-3 bg-roadmap-blue-light"
+        aria-hidden={true}
+      />
+      <div className="dashboard-opportunities-list desktop:margin-right-1">
+        <div className="desktop:padding-right-105" data-testid="visible-opportunities">
+          {certificationsUnlocked() ? (
+            nonHiddenSortedCertifications.map((cert) => (
+              <OpportunityCard key={cert.id} opportunity={cert} urlPath="certification" />
+            ))
+          ) : (
+            <></>
+          )}
+          {/* {nonHiddenSortedFundings.map((funding) => (
+            <OpportunityCard key={funding.id} opportunity={funding} urlPath="funding" />
+          ))} */}
+          {/* {nonHiddenSortedFundings.length + nonHiddenSortedCertifications.length === 0 && (
+            <div className="fdc fac margin-y-3">
+              <h3 className="text-normal">{Config.dashboardDefaults.emptyOpportunitiesHeader}</h3>
+              <img src={`/img/signpost.svg`} className="" alt="" />
+              <p className="text-center">{Config.dashboardDefaults.emptyOpportunitiesText}</p>
+            </div>
+          )} */}
```

Comment:
there's a lot of comments in this file about the fundings. What's going on? Will this code work for our existing Oscar experience for fundings?